src/legedit2/helpers/ProjectHelper.java

```diff
@@ -182,127 +182,156 @@ public static void newProject()
 	
 	private static void parseDeck(Node node)
 	{
-		Deck deck = new Deck();
-		
-		if (node.getAttributes().getNamedItem("name") != null)
-		{
-			deck.setName(node.getAttributes().getNamedItem("name").getNodeValue());
-		}
-		
-		if (node.getAttributes().getNamedItem("template") != null)
-		{
-			for (DeckType type : DeckType.getDeckTypes())
+        Deck deck = new Deck();
```

Comment:
Indentation issue here. Perhaps a difference between spaces and tabs? Ditto for line 186.

P.S. I’m a nobody here. Just a Watcher of this project....

Reply:
Thx DarthEd77, yes it was a space vs tab issue. Looks like I fixed it directly in the pull request but aren't super sure so I'll keep an eye on it to make sure it did update it correctly.

PS: In the grand scheme of the internet, we are all nobodys :)